antlr-kotlin-runtime/src/commonMain/kotlin/org/antlr/v4/kotlinruntime/ANTLRInputStream.kt

```diff
@@ -43,8 +42,9 @@ open class ANTLRInputStream : CharStream {
     constructor() {}
 
     /** Copy data in string to a local char array  */
+    
     constructor(input: String) {
-        this.data = input.toCharArray()
+        this.data = input.map { it }.toCharArray()
```

Comment:
Mmm, suspicious

Reply:
Collection<Char>.toCharArray() is defined by Kotlin Common Stdlib. String.toCharArray() is also defined but has experimental state.

Reply:
please reuse the extension method we discussed